src/main/java/frc/robot/subsystems/grabber/GrabberInputs.java

```diff
@@ -0,0 +1,15 @@
+package frc.robot.subsystems.grabber;
+
+import edu.wpi.first.epilogue.Logged;
+
+@Logged
+public class GrabberInputs {
+    public double currentVelocity;
+    public double currentAppliedVoltage;
+    public double currentPosition;
```

Comment:
I don't think position is helpful to us here. We don't care about the position of the motor, just the velocity.